proto/viam/service/vision/v1/vision.proto

```diff
@@ -176,6 +183,31 @@ message GetPropertiesRequest {
   google.protobuf.Struct extra = 99;
 }
 
+message CaptureAllFromCameraRequest {
+  // name of the vision service
+  string name = 1;
+  // name of camera source to use as input
+  string camera_name = 2;
+  // whether or not including the image in the response
+  bool return_image = 3;
+  // whether or not including classifications in the response
+  bool return_classifications = 4;
+  // whether or not including detections in the response
+  bool return_detections = 5;
+  // whether or not including pcd in the response
+  bool return_object_point_clouds = 6;
+
+  google.protobuf.Struct extra = 99;
+}
+
+message CaptureAllFromCameraResponse {
+  component.camera.v1.Image image = 1;
+  repeated Detection detections = 2;
+  repeated Classification classifications = 3;
+  repeated common.v1.PointCloudObject objects = 4;
+  google.protobuf.Struct extra = 99;
```

Comment:
I don't think `extra` needs to be included in the response?

Reply:
It's actually in the scope for the message
https://docs.google.com/document/d/119ITK_p0xKyYBTlyksYUEjsHm_lAx3b4c9CaJNtdOnI/edit#heading=h.tcicyojyqi6c